src/Tizen.Wearable.CircularUI.Forms/CircleStepper.cs

```diff
@@ -49,27 +49,62 @@ public class CircleStepper : Xamarin.Forms.Stepper, IRotaryFocusable
         /// <since_tizen> 4 </since_tizen>
         public static readonly BindableProperty TitleProperty = BindableProperty.Create(nameof(Title), typeof(string), typeof(CircleStepper), null);
 
+        /// <summary>
+        /// BindableProperty. Identifies whether min/max value is wrapped or not.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty IsWrapEnabledProperty = BindableProperty.CreateAttached(nameof(IsWrapEnabled), typeof(bool), typeof(CircleStepper), true);
```

Comment:
Why don't you use `Create()` instead? It seems that `IsWrapEnabled` is set on the `CircleStepper` itself.

Reply:
done. change CreateAttached to Create